uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/BAO/AbsenceTypeTest.php

```diff
@@ -36,7 +36,23 @@ public function testTypeTitlesShouldBeUnique() {
       'CRM_HRLeaveAndAbsences_Exception_InvalidAbsenceTypeException',
       'Absence Type with same title already exists!'
     );
-    AbsenceTypeFabricator::fabricate(['title' => 'Type 1']);
+    AbsenceTypeFabricator::fabricate(['title' => ' Type 1 ']);
+  }
+
+  public function testTypeTitleShouldNotBeEmpty() {
+    $params = ['', '   '];
```

Comment:
Should be named like `$possibleValues` -> `$value`

Reply:
I think `params` is also descriptive enough as those are test parameters 🤷‍♂️

Reply:
Concerning all cases listed in $params, if any of them should fail, then jenkins will not pass for that test.

Reply:
Ok, let it be $params.